src/Contrib/OtlpGrpc/Exporter.php

```diff
@@ -69,6 +70,18 @@ public function __construct(
             $this->getIntFromEnvironment(Env::OTEL_EXPORTER_OTLP_TIMEOUT, $timeout);
 
         $this->setSpanConverter(new SpanConverter());
+        $this->setRetryPolicy(ExponentialWithJitterRetryPolicy::getDefault());
```

Comment:
the default retry policy has 5 max attempts - didn't we discuss in SIG that by default there would not be a retry policy (or a retry policy that didn't retry) ?